tests/end2end/features/thumbnail/thumbnaillazy.feature

```diff
@@ -0,0 +1,66 @@
+Feature: Lazy load thumbnails
+
+    Scenario: Eager loading (default behavior)
+        Given I open 10 images without leading zeros in their name
+        And I enter thumbnail mode
+        When I run 5scroll right
+        Then there should be 10 rendered thumbnails
+        And the first index should be 0
+        And the last index should be 9
+		And thumbnails should load in order of closeness to selected
```

Comment:
Tabs instead of spaces I guess? Just leaving this here before I forget, will continue understanding the behaviour and leave an overall comment.